Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM maven:3-openjdk-11 AS build
+
+# Copy OpenSRP Source
+COPY . /tmp/opensrp-server-web
+
+# Build WAR file
+ARG opensrp_maven_package_profiles="postgres,jedis,oauth2"
+
+RUN mvn clean package -Dmaven.test.skip=true -P $opensrp_maven_package_profiles -f /tmp/opensrp-server-web/pom.xml
+
+# Explode WAR file
+WORKDIR /tmp/opensrp-server-web-exploded && \
+RUN jar -xvf /tmp/opensrp-server-web/target/opensrp.war
```

Comment:
why are we exploding the war file, tomcat should be able to explode during startup

Reply:
It shaves off some time needed to extract the war file when starting the container, and start serving requests immediately. This is extremely helpful when dynamically scaling to meet application demand.